README.md

```diff
@@ -84,7 +84,18 @@ namespace TwitchLib.EventSub.Websockets.Test
 
             if (!e.IsRequestedReconnect)
             {
-                // subscribe to topics
+                /*
+                Subscribe to topics via the TwitchApi.Helix.EventSub object, this example shows how to subscribe to the channel follow event used in the example above.
```

Comment:
I would add for dull transparency that twitching.api needs to be installed to access this class of objects.

Reply:
Are you/someone else able to add that as I dont know what your expected release/consumption model is for this library, I just include the `TwitchLib` nuget library which contains all of them. So you would probably be in a better position to recommend how you want people to consume the library to use with this.

Reply:
I usually don't use the suite myself since I like to streamline. But this would help with confusion for people that use individual repos and not have API already installed. 

Basically say Prerequisite = Twitchlib.api

Reply:
Added what you asked, let me know if you want any other alterations.